core/integration/engine_test.go

```diff
@@ -13,6 +13,8 @@ import (
 	"github.com/stretchr/testify/require"
 )
 
+const alpineImage = "alpine:3.18.2"
```

Comment:
We seem to define the patch version here, but not in `internal/mage/util/engine.go`. I am all for patch pinning, but sticking to minor works too. Doesn't matter much as long as we are consistent.

Also, would it be worth declaring this version in a single place? OK not to do it it's not an easy change, this is already a great improvement as is!

Reply:
> Also, would it be worth declaring this version in a single place? OK not to do it it's not an easy change, this is already a great improvement as is!

this is used for testing and the `util/engine.go` is used to assemble the engine dev container. It's two different variables intentionally.